sympy/solvers/polysys.py

```diff
@@ -185,6 +194,16 @@ def solve_generic(polys, opt):
     .. [Cox97] D. Cox, J. Little, D. O'Shea, Ideals, Varieties
     and Algorithms, Springer, Second Edition, 1997, pp. 112
 
+    Raises
+    ========
+
+    NotImplementedError
+        The system is not zero-dimensional (does not have a finite number of solutions)
+
+        ``strict`` is True and at least one polynomial in the Groebner basis does not
```

Comment:
Put "If" at the beginning of each of these sentences. Although they aren't really proper sentences they should still be ended with a full stop. Also these lines should be wrapped.